src/transform/src/join_implementation.rs

```diff
@@ -611,20 +611,19 @@ mod delta_queries {
             )?;
 
             // Count new arrangements.
-            let new_arrangements: usize =
-                orders
-                    .iter()
-                    .flat_map(|o| {
-                        o.iter().skip(1).filter_map(|(c, key, input)| {
-                            if c.arranged {
-                                None
-                            } else {
-                                Some((input, key))
-                            }
-                        })
+            let new_arrangements: usize = orders
+                .iter()
+                .flat_map(|o| {
+                    o.iter().skip(1).filter_map(|(c, key, input)| {
+                        if c.arranged() {
+                            None
+                        } else {
+                            Some((input, key))
+                        }
                     })
-                    .collect::<BTreeSet<_>>()
-                    .len();
+                })
+                .collect::<BTreeSet<_>>()
+                .len();
```

Comment:
(Just changed `c.arranged` to `c.arranged()`, but `cargo fmt` reformatted the whole block.)

Reply:
🙃